crates/bevy_text/src/text2d.rs

```diff
@@ -107,7 +107,7 @@ pub struct QueuedText2d {
     entities: Vec<Entity>,
 }
 
-/// Updates the TextGlyphs with the new computed glyphs from the layout
+/// Updates the `TextGlyphs` with the new computed glyphs from the layout
```

Comment:
We don't have a type named `TextGlyphs` (or `TextGlyph`). Not sure if this documentation is outdated?

Reply:
Also referenced at [`crates/bevy_ui/src/widget/text.rs:37`](https://github.com/bevyengine/bevy/blob/601cc0cbe3dc73c0db1993184f4ec47783e83884/crates/bevy_ui/src/widget/text.rs#L37)